openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -937,6 +931,11 @@ def process(self, instance):
                 self.deadline_url = instance.data.get("deadlineUrl")
             assert self.deadline_url, "Requires Deadline Webservice URL"
 
+            if self.deadline_priority:
+                render_job["Props"]["Pri"] = self.deadline_priority
+            else:
+                render_job["Props"]["Pri"] = instance.data.get("priority")
+
```

Comment:
I know this matches older code but this does seem odd.

Instead of changing the Priority of the job that we are submitting we're changing the properties data of the already submitted job to update how we're setting the priority for the Publish Job.

It feels like we should be moving this to [around line 245](https://github.com/pypeclub/OpenPype/blob/700184cfdc2f3dbabb3b49bbc1d9464fd3939a25/openpype/modules/deadline/plugins/publish/submit_publish_job.py#L245) and actually just set:
```python
            if self.deadline_priority:
                priority = self.deadline_priority
            else:
                priority = instance.data.get("priority")
```

And pass that to `priority` on [line 257](https://github.com/pypeclub/OpenPype/blob/700184cfdc2f3dbabb3b49bbc1d9464fd3939a25/openpype/modules/deadline/plugins/publish/submit_publish_job.py#L257).